07-routing-gossip.md

```diff
@@ -1,19 +1,25 @@
 # BOLT #7: P2P Node and Channel Discovery
 
-This specification describes simple node discovery, channel discovery and channel update mechanisms which do not rely on a third-party to disseminate the information.
+This specification describes simple node discovery, channel discovery, and channel update mechanisms that do not rely on a third-party to disseminate the information.
 
 Node and channel discovery serve two different purposes:
 
- - Channel discovery allows the creation and maintenance of a local view of the network's topology such that the node can discover routes to the desired destination.
- - Node discovery allows nodes to broadcast their ID, host and port, such that other nodes can open connections and establish payment channels.
+ - Channel discovery allows the creation and maintenance of a local view of the network's topology, so that the node can discover routes to the desired destination.
+ - Node discovery allows nodes to broadcast their ID, host, and port, so that other nodes can open connections and establish payment channels.
  
-Peers in the network exchange `channel_announcement` messages that contain information about new channels between two nodes.  They can also exchange `node_announcement` messages which supply additional information about nodes, and `channel_update` messages which update information about a channel.
+To support channel discovery, peers in the network exchange
+`channel_announcement` messages, which contain information about new
+channels between two nodes. They can also exchange `channel_update`
+messages, which update information about a channel. There can only be
+one valid `channel_announcement` for any channel, but at least two
+`channel_update` messages are expected.
 
-There can only be one valid `channel_announcement` for any channel,
-but multiple `node_announcement` messages are possible (to update node
-information), and at least two `channel_update` messages are expected.
+To support node discovery, peers exchange `node_announcement`
+messages, which supply additional information about nodes There can be
```

Comment:
Missing period between "about nodes" and "There can be"

Reply:
Fixed.